File: porovnavani_textu.cpp
#include<iostream>
#include<string>
std::string max, x;
int main(){
    std::cin >> max;
    for(int i = 0;i<4;i++){
        std::cin >> x;
        if(x>max){
            max = x;
        }
    }
std::cout<<max;
}

File: porovnavani_textu.pas
program Hello;
var
  x, max: string;  
  i: integer;      
begin
  readln(max);     
  for i := 1 to 9 do 
  begin
    readln(x);     
    if x > max then
      max := x;    
  end;
  writeln('nejvetsi je ', max);  
end.
